### PR TITLE
feat(injections)!: switch to upstream format

### PR DESCRIPTION
**Pull request by clason**
* Raise minimum Nvim version to v0.9.1
* Switch to upstream injection format
* Backport new injections from `main`

### CODE REVIEW

**Review by amaanq — APPROVED** [review]

**Comment by ObserverOfTime** [comment]
Missing the downcase fix.

**Comment by clason** [comment]
This does merit a proper review for the injection changes; tests caught a few, and the added `awk` injection looks suspicious @lucario387

**Comment by ObserverOfTime** [comment]
That should be in bash. :thinking:

**Comment by clason** [comment]
But bash already has (different) injections?

File: queries/awk/injections.scm [hunk 1]
```diff
@@ -1,2 +1,12 @@
-(comment) @comment
-(regex) @regex
+((comment) @injection.content
+ (#set! injection.language "comment"))
+
+((regex) @injection.content
+ (#set! injection.language "regex"))
+
+((redirected_io_statement
```
Comment (lucario387) [inline queries/awk/injections.scm:7]: Wait when did this appear 😆 . I have no idea of this

**Comment by ObserverOfTime** [comment]
It was misplaced in 1d36ff0023a82185c0ca5110e44859f3005d467a

**Comment by ObserverOfTime** [comment]
Now it's just gone? :thinking:

**Comment by clason** [comment]
Yeah, I see what's gone wrong (the PR on `main` changed the wrong file, and we aren't running CI on that branch yet).

**Review by ObserverOfTime — APPROVED** [review]

**Review by lucario387 — APPROVED** [review]

**Review by lucario387 — APPROVED** [review]
I think it's fine

File: queries/awk/injections.scm (hunk 1 above)
Comment (clason) [reply queries/awk/injections.scm]: No idea; cropped up when rebasing. Glitch in the git?

**Comment by clason** [comment]
YOLO. Brace for complaint.

**Pull request merged**